packages/inventory/src/redux/entities.js

```diff
@@ -233,6 +240,7 @@ export default {
     [ACTION_TYPES.ALL_TAGS_PENDING]: (state) => ({ ...state, allTagsLoaded: false }),
     [ACTION_TYPES.LOAD_ENTITIES_PENDING]: entitiesPending,
     [ACTION_TYPES.LOAD_ENTITIES_FULFILLED]: entitiesLoaded,
+    [ACTION_TYPES.LOAD_ENTITIES_REJECTED]: entitiesLoadingRejected,
```

Comment:
Should we also add it for `LOAD_TAGS` action? 🤔

Reply:
Should we? In compliance we don't use the global filter yet, so I'm not sure how that deals with rejection, but if it also causes the inventory to get stuck in the loading state it should definitely be added.

Reply:
Yup, good point. If consumer wants to show tags in table and the request fails it would cause the table to be stuck in infinit loading.

Reply:
So also for `ALL_TAGS_REJECTED`.